opensrp-app/src/main/java/org/smartregister/dao/AbstractDao.java

```diff
@@ -0,0 +1,217 @@
+package org.smartregister.dao;
+
+import android.database.Cursor;
+
+import net.sqlcipher.database.SQLiteDatabase;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.Nullable;
+import org.smartregister.repository.Repository;
+import org.smartregister.view.activity.DrishtiApplication;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import timber.log.Timber;
+
+public class AbstractDao {
+
+    private static SimpleDateFormat DOB_DATE_FORMAT;
+    private static SimpleDateFormat NATIVE_FORMS_DATE_FORMAT;
+    private static Repository repository;
+
+    public static SimpleDateFormat getDobDateFormat() {
+        if (DOB_DATE_FORMAT == null)
+            DOB_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd", Locale.getDefault());
```

Comment:
I would suggest this to be changed to Locale.English. It will probably fail if Locale.getDefault() return `ar` instead of `en`. Locale.getDefault() would be good if you are parsing Date for display purposes.